cmd/notation/sign.go

```diff
@@ -47,14 +47,17 @@ func signCommand(opts *signOpts) *cobra.Command {
 		Short: "Sign artifacts",
 		Long: `Sign artifacts
 
-Prerequisite: a signing key needs to be configured using the command "notation key".
+Note: a signing key needs to be configured using the command "notation key" or with specifying a key ID
```

Comment:
How about:
```suggestion
Recommend: configure signing key using the command "notation key add".
```

Reply:
@patrickzheng200 I would suggest `Note: a signing key needs to be configured using the command "notation key add" or with specifying a key ID`. There is an example to sign with a specified key ID. It doesn't require `notation key add`.

Reply:
I'm not sure if this is the best phrasing, since using a key ID is not configuring a new key. What about `Note: a signing key must be specified. This can be done temporarily by specifying a key ID, or a new key can be configured using the command "notation key add"`

Reply:
> I'm not sure if this is the best phrasing, since using a key ID is not configuring a new key. What about `Note: a signing key must be specified. This can be done temporarily by specifying a key ID, or a new key can be configured using the command "notation key add"`

Yeah, I think Kody's suggestion is a bit better here. @FeynmanZhou

Reply:
Agree with @kody-kimberl . Updated.